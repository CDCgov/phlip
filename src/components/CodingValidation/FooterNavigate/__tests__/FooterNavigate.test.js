import React from 'react'
import { shallow } from 'enzyme'
import { FooterNavigate } from '../index'

const props = {
  currentIndex: 0,
  getNextQuestion: () => {},
  getPrevQuestion: () => {},
  totalLength: 1
}

describe('Coding -- FooterNavigate component', () => {
  test('should render correctly', () => {
    expect(shallow(<FooterNavigate {...props} />)).toMatchSnapshot()
  })
})
