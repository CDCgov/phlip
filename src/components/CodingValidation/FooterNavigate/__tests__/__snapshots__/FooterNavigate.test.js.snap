// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Coding -- FooterNavigate component should render correctly 1`] = `
<Row
  component={
    <WithStyles
      item={true}
    />
  }
  displayFlex={true}
  flex={false}
  style={
    Object {
      "alignItems": "center",
      "alignSelf": "flex-end",
      "height": 50,
      "justifyContent": "space-between",
      "paddingTop": 10,
    }
  }
/>
`;
